src/resolve-styles.js

```diff
@@ -62,17 +62,26 @@ const _resolveChildren = function({
     };
   }
 
+  const shouldResolveStyles = function(element) {
+    return (element.type && !element.type.__isRadiumEnhanced);
+  };
+
   if (React.Children.count(children) === 1 && children.type) {
     // If a React Element is an only child, don't wrap it in an array for
     // React.Children.map() for React.Children.only() compatibility.
     const onlyChild = React.Children.only(children);
-    return resolveStyles(component, onlyChild, config, existingKeyMap);
+
+    if (shouldResolveStyles(onlyChild)) {
```

Comment:
nice! We should also add this every else we recursively call resolveStyles; alternatively, add it to resolveStyles itself, add a flag to skip the check, and only skip the check in the exported function.